engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -797,7 +798,7 @@ private void execute() throws YarnException, IOException
       }
 
       numTotalContainers += containerRequests.size();
```

Comment:
If numTotalContainers is needed, it needs to be defined (what is the meaning of the number and how it helps?), otherwise, if it is not needed, it needs to be removed.